thingif/src/test/java/com/kii/thingif/clause/trigger/AndClauseTest.java

```diff
@@ -0,0 +1,50 @@
+package com.kii.thingif.clause.trigger;
+
+import com.kii.thingif.SmallTestBase;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class AndClauseTest extends SmallTestBase {
+
+    @Test
+    public void testToJSONObject() throws JSONException{
+        JSONObject[] clauses1 = {
+                RangeClauseInTrigger.greaterThan("alais", "f", 23).toJSONObject(),
+                RangeClauseInTrigger.lessThan("alais", "f", 230).toJSONObject()
+        };
+        JSONObject[] clauses2 = {
+                new NotEqualsClauseInTrigger(new EqualsClauseInTrigger("alias", "f", 46)).toJSONObject(),
+                RangeClauseInTrigger.greaterThan("alias", "f", 23).toJSONObject()
+        };
+        JSONObject[] clauses3 = {
+                new NotEqualsClauseInTrigger(new EqualsClauseInTrigger("alias", "f", 100)).toJSONObject(),
+                RangeClauseInTrigger.range("alias", "f", 23, false, 230, false).toJSONObject()
+        };
+        JSONObject[] expectedJsons = {
+                new JSONObject().put("type", "and").put("clauses", clauses1),
+                new JSONObject().put("type", "and").put("clauses", clauses2),
+                new JSONObject().put("type", "and").put("clauses", clauses3)
+        };
```

Comment:
I think that creating expected data with test target classes is not good.

What do you think?